CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
 # osquery Changelog
 
+<a name="5.4.0"></a>
+## [5.4.0](https://github.com/osquery/osquery/releases/tag/5.4.0)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.3.0...5.4.0)
+
+Representing commits from 15 contributors! Thank you all.
+
+### New Features
+
+- We're extending macOS Endpoint Security to include File Integrity monitoring. Check out the new `es_process_file_events` table. ([#7579](https://github.com/osquery/osquery/pull/7579))
+- Add Docker build scripts and configuration ([#7619](https://github.com/osquery/osquery/pull/7619))
+
+### Deprecation Notices
+
+- Prevent CLI_FLAGs to be set via config ([#7561](https://github.com/osquery/osquery/pull/7561))
+- Remove the `lldp_neighbors` table ([#7664](https://github.com/osquery/osquery/pull/7664))
+
+
+
+### Table Changes
+
+- New Table: `es_process_file_events` for macOS Endpoint Security based FIM ([#7579](https://github.com/osquery/osquery/pull/7579))
+- New Table: `password_policy` table for macOS ([#7594](https://github.com/osquery/osquery/pull/7594))
+- New Table: `windows_update_history` ([#7407](https://github.com/osquery/osquery/pull/7407))
```

Comment:
All of these new tables maybe should go up in the `New Features` section?

Reply:
We don't usually put new tables into new features. 

I thought it would be nice to advertise the ES support, but it's always a toss up.